doc/book/cookbook/route-specific-pipeline.md

```diff
@@ -62,7 +62,7 @@ use Zend\ServiceManager\DelegatorFactoryInterface;
 use Zend\ServiceManager\ServiceLocatorInterface;
 use Zend\Stratigility\MiddlewarePipe;
 
-class ApiResourcePipelineDelegatorFactory
+class ApiResourcePipelineDelegatorFactory implements DelegatorFactoryInterface
```

Comment:
it is better to use `DelegatorFactoryInterface` from `ServiceManager\Factory` namespace so we use : 
```php
public function __invoke(
      ContainerInterface $container,
      $name,
      callable $callback,
      array $options = null
)
```